### DISCUSSION TITLE
Single node, cross pod network working after `iptables -P FORWARD ACCEPT`, Why ?

### DISCUSSION THREAD

**Discussion by wenerme** [k3s-io/k3s#9852] [Q&A]
**Environmental Info:**
K3s Version:  v1.28.8+k3s1

Node(s) CPU architecture, OS, and Version:  

```
6.6.23-0-lts #1-Alpine SMP PREEMPT_DYNAMIC Thu, 28 Mar 2024 11:27:43 +0000 x86_64 GNU/Linux
```

Cluster Configuration:

single

**Describe the bug:**

after upgrade to alpine 3.19 (the iptable default to nft), k3s cross pod network and external network not working ,using default flannel config, after `iptables -P FORWARD ACCEPT`,  network works again.

**Steps To Reproduce:**

**Expected behavior:**

cross pod should just work

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment]
Do you have a host-based firewall enabled? firewalld, ufw, or so on? Do you have something else configured on your node, such that the default iptables policy is `DROP` instead of `ACCEPT`?

**Comment by wenerme** [comment]
@brandond 
AlpineLinux, no ufw, no iptables rules for fw, but I do have multi intferface.

I have eth0, eth4, eth5, flannel choose eth4, I think maybe this is the problem.
